### PR TITLE
tcc: add page

### PR DESCRIPTION
**Pull request by kenbyte**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
Edit: formatting

### CODE REVIEW

File: pages/common/tcc.md
Comment (kbdharun) [inline pages/common/tcc.md:3]:
```suggestion
> A tiny C compiler that can run C source files as scripts and otherwise has command line options similar to `gcc`.
```

File: pages/common/tcc.md
Comment (kbdharun) [inline pages/common/tcc.md:8]:
```suggestion
`tcc -o {{executable_name}} {{path/to/a.c}} {{path/to/b.c}}`
```
Note: `{{}}` used here is used for highlighting placeholders in our clients.

File: pages/common/tcc.md
Comment (kbdharun) [inline pages/common/tcc.md:10]:
```suggestion
- Directly run `a.c` like a script and pass arguments to it:
```

File: pages/common/tcc.md
Comment (kbdharun) [inline pages/common/tcc.md:12]:
```suggestion
`tcc -run {{path/to/a.c}} {{arguments}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

File: pages/common/tcc.md
Comment (kbdharun) [inline pages/common/tcc.md:6]:
```suggestion
- Compile and link `a.c` and `b.c` to generate an executable:
```
How about this?

**Comment by kenbyte** [comment]
Thanks for the review; I've applied your suggestions.

File: pages/common/tcc.md
Comment (EmilySeville7cfg) [inline pages/common/tcc.md:8]: Converter TlDr -> CLIP (Command Line Interface Pages) can't recognize any of `{{path/to/a.c}} {{path/to/b.c}}` placeholders. To fix this issue add `_file` suffix right before .c extension.

Note: issue was fixed manually [here](https://github.com/command-line-interface-pages/cli-pages/pull/47).

Comment (kbdharun) [reply pages/common/tcc.md:8]: Or it can be replace with `file1` and `file2` instead of `a` and `b`.

Comment (EmilySeville7cfg) [reply pages/common/tcc.md:8]: Sure, or with `{{path/to/source_file.c}}`.

Comment (kenbyte) [reply pages/common/tcc.md]: I've now implemented those suggestions.

**Comment by EmilySeville7cfg** [comment]
I can't approve this PR to make it mergeable, I have no rights here, while @kbdharun, has.

**Comment by kenbyte** [comment]
Hi, just following up to ask if there's anything further that I need to do in order to get this merged

**Comment by kbdharun** [comment]
> Hi, just following up to ask if there's anything further that I need to do in order to get this merged

No, once it gets 2 approvals it will get merged, I will approve it now.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kenbyte** [comment]
Just waiting for the final review

File: pages/common/tcc.md
Comment (CleanMachine1) [inline pages/common/tcc.md:10]:
```suggestion
- Directly run an input file like a script and pass arguments to it:
```

File: pages/common/tcc.md
Comment (CleanMachine1) [inline pages/common/tcc.md:16]: This doesn't match the TLDR syntax.

File: pages/common/tcc.md
Comment (CleanMachine1) [inline pages/common/tcc.md:6]:
```suggestion
- Compile and link 2 source files to generate an executable:
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/tcc.md
Comment (kenbyte) [reply pages/common/tcc.md:16]: The alternative is to remove this part for not being a terminal command, but surely that's worse than leaving it in as valuable information. I can remove it if you guys really want me to, but I feel that that would be needlessly prohibitive without providing any actual benefit. Especially since this is a useful and unique differentiating feature of `tcc`

Comment (CleanMachine1) [reply pages/common/tcc.md:16]: Ok, sure, we will leave it.

**Comment by CleanMachine1** [comment]
Thanks!

**Pull request merged**
